frontend/src/components/common/Table.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import {
+  TableContainer,
+  Table as T,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+} from "@chakra-ui/react";
+
+export interface TableRow {
+  values: string[];
+  menu: React.ReactElement;
+}
+
+interface TableProps {
+  tableHeaders: string[];
+  tableRows: TableRow[];
+}
+
+export const Table = ({
+  tableHeaders,
+  tableRows,
+}: TableProps): React.ReactElement => {
+  return (
+    <TableContainer
+      padding="0.5em"
+      border="1px solid"
+      borderColor="#E2E8F0"
+      borderRadius="12px"
+      minWidth="100%"
+    >
```

Comment:
I couldn't figure out how to extract the css here to a theming file... I think it might not be possible because `TableContainer` is essentially just a `div`, but let me know if this is not the case